### PR TITLE
Feature/language change

### PR DESCRIPTION
**Pull request by Ayerbe88**
Language change visuals will change, how it is now (two flag icons) is to be able to test and show functionality of changing language, which is what it is about along with the communication with the web component. When we have advanced the task of creating the dropdown for user options, we'll move this functionality to an option from that list and remove the language icons.

### CODE REVIEW

File: am-ui/src/app/components/home/home.component.ts [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+
+  constructor(private translate: TranslateService) {}
+
+  ngOnInit(): void {
+    this.subscription = this.translate.onLangChange.subscribe(appLanguage => {
+      this.appLanguage = appLanguage.lang;
+    });
+  }
+
+  ngOnDestroy() {
```
Comment (stefanleminh) [inline am-ui/src/app/components/home/home.component.ts:22]: Might as well implement the OnDestroy interface here (similar to how it already implements OnInit) to get rid of this warning

File: webcomponents/search-webcomponent/src/app/mp-search.component.ts [hunk 2]
```diff
@@ -26,13 +33,20 @@ export class MpSearchComponent implements OnInit {
...
   constructor(
-    private readonly translate: TranslateService,
-    private store: Store<MarketplaceState>
-  ) {
-    translate.setDefaultLang('de');
+    private store: Store<MarketplaceState>,
+    private localization: LocalizationService
+  ) { }
+
+  ngOnChanges(changes: SimpleChanges) {
```
Comment (stefanleminh) [inline webcomponents/search-webcomponent/src/app/mp-search.component.ts:45]: Might as well implement the OnChanges interface to get rid of this warning, similar to previous issue.

**Review by stefanleminh — CHANGES_REQUESTED** [review]

File: am-ui/src/app/components/home/home.component.ts (hunk 1 above)
Comment (Ayerbe88) [reply am-ui/src/app/components/home/home.component.ts:22]: True, also I think might not go correctly through life cycle if not done under implements

File: webcomponents/search-webcomponent/src/app/mp-search.component.ts (hunk 2 above)
Comment (Ayerbe88) [reply webcomponents/search-webcomponent/src/app/mp-search.component.ts:45]: done

**Review by stefanleminh — APPROVED** [review]

**Pull request merged**